src/NGPL.xml

```diff
@@ -0,0 +1,95 @@
+<spdx name="Nethack General Public License" identifier="NGPL" osi-approved="true">
+  <urls>
+    <url>http://www.opensource.org/licenses/NGPL</url>
+  </urls>
+  <license>
+    <title>
+      <p>NETHACK GENERAL PUBLIC LICENSE</p>
+    </title>
+    <copyright>
+      <p>(Copyright 1989 M. Stephenson)</p>
+    </copyright>
+    <optional>
```

Comment:
I'd support making this optional; it seems to be a note and doesn't seem to have any legal effect. Not sure if there is an official process for marking something optional in a template though.